checkstyle-tester/checks-nonjavadoc-error.xml

```diff
@@ -331,6 +331,7 @@
     <module name="SuperClone"/>
     <module name="SuperFinalize"/>
     <module name="UnnecessaryParentheses"/>
+    <module name="UnnecessarySemicolonAfterOuterTypeMemberDeclaration"/>
```

Comment:
PR title is `UnnecessarySemicolonAfterOuterTypeDeclarationCheck`. There is no "Member" in its name.

Reply:
Done